.cirrus.yaml

```diff
@@ -0,0 +1,52 @@
+env:
+  # Allows to run builds for the 50 last commits in a branch:
+  CIRRUS_CLONE_DEPTH: 50
+  CIRRUS_SHELL: bash
```

Comment:
Little improvement I see: set CIRRUS_SHELL on the task level; build_linux_task could execute with bash, and build_windows_task could use powershell by default.